doc/sphinx/practical-tools/utilities.rst

```diff
@@ -124,22 +124,28 @@ avoid putting your code into subdirectories such as "src" unless you want
 that to be part of the :term:`logical path` in Coq.
 
 For example, here's a minimal _CoqProject file for the `MyProject` package
-(the logical name of the package), which has two script files `File1.v`
-and `SubDir/File2.v` whose logical paths
-are `MyProject.File1` and `MyProject.SubDir.File2`.  `_CoqProject` should be
-in the top-level directory of your project.  We recommend naming that directory
-`MyProject`, the same as the logical name::
+(the logical name of the package), which includes all the ``.v`` files (and
+other file types) in the current directory and its subdirectories:
+
+  -R . MyPackage
+  .
+
+You can list multiple directories if you wish.
+
+In addition, you can list individual files, for example for the two script files
+`File1.v` and `SubDir/File2.v` whose logical paths are `MyProject.File1` and
+`MyProject.SubDir.File2`.  We recommend naming that directory `MyProject`, the
+same as the logical name::
 
   -R . MyPackage
   File1.v
   SubDir/File2.v
 
 :n:`-R . MyPackage` (see :ref:`here <-Q-option>`) declares that all `.v` files in the directory containing
 `_CoqProject` or any subdirectory are part of the package `MyPackage`.  We recommend
-using a single `-R` with `.` as the :term:`physical path` for simplicity.  It's
-also necessary to (redundantly) list every `.v` file that's part of the package.
-Perhaps this will be simplified in the future.  The order of the entries does not
-matter.
+using a single `-R` with `.` as the :term:`physical path` for simplicity.  You must also
+identify all the `.v` files in your package.  We recommend using directory names
```

Comment:
I am wary of all these recommendations. That is certainly not something I would ever recommend to a user. And in fact, this is not even the common practice in the Coq ecosystem. Common practice is to have a directory `theories` and to have the corresponding option `-R theories Logical`.